client/src/components/FriendsDashboard/Suggestions.js

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components';
+import { useState, useEffect, useContext } from 'react';
+import SearchArea from './SearchArea';
+import axios from 'axios';
+
+import List from '@material-ui/core/List';
+
+import User from './User';
+import UserContext from '../../context/UserContext';
+
+export default function Suggestions () {
+    const { userData } = useContext(UserContext);
+    
+    const [users, setUsers] = useState('');
+    const [query, setQuery] = useState('');
+
+    useEffect(() => {
+        const getUsers = async () => {
+            try {
+                await axios.get('http://localhost:5000/friends/usersList')
+                    .then(
+                        resp => {
+                            const filtered = resp.data.user.filter((user) => user._id !== userData.user.id)
```

Comment:
We can avoid a frontend filter by moving this logic to the backend (the db and the server is more optimized to handle an operation like this). Filter the list out before returning it to the frontend